crystal_diffusion/models/score_network.py

```diff
@@ -221,13 +253,24 @@ def _forward_unchecked(self, batch: Dict[AnyStr, torch.Tensor]) -> torch.Tensor:
         times = batch[TIME].to(relative_coordinates.device)  # shape [batch_size, 1]
         input = torch.cat([self.flatten(relative_coordinates), times], dim=1)
 
-        output = self.mlp_layers(input).reshape(relative_coordinates.shape)
+        forces_input = self.condition_embedding_layer(self.flatten(batch[CARTESIAN_FORCES]))
+
+        output = input
+        for i, (layer, condition_layer) in enumerate(zip(self.mlp_layers, self.conditional_layers)):
+            if i != 0:
```

Comment:
In the original code, only the **last layer** had no ReLU. Here, it seems that the **first layer** has no ReLU. Is this intentional?
Doesn't a ReLU on the last layer force all the outputs to be non-negative?